src/main.css

```diff
@@ -120,11 +120,11 @@ input[type=range]::-moz-range-thumb {
   width: 10px;
 }
 
-input[type=range]:focus {
-  outline: 01px solid white;
+input[type="range"]:focus {
+  outline: 1px solid white;
```

Comment:
We actually have a var for `--white`. Not sure if we should use that globally or not (so I left it).